plugins/BEdita/Core/tests/TestCase/Model/Entity/ObjectTypeTest.php

```diff
@@ -135,39 +135,39 @@ public function testSetName()
     }
 
     /**
-     * Data provider for `testGetSetPluralized` test case.
+     * Data provider for `testGetSetSingular` test case.
      *
      * @return array
      */
-    public function getSetPluralizedProvider()
+    public function getSetSingularProvider()
     {
         return [
-            'default' => ['foo_bars', 'foo_bar', 'FooBars'],
-            'missing' => ['foo_bars', 'foo_bar', null],
+            'default' => ['foo_bar', 'foo_bars', 'FooBar'],
+            'missing' => ['foo_bar', 'foo_bars', null],
         ];
     }
 
     /**
-     * Test getter/setter method for `pluralized`.
+     * Test getter/setter method for `singular`.
      *
      * @param string $expected Expected result.
      * @param string $name Object type name.
-     * @param string|null $pluralized Object type pluralized name.
+     * @param string|null $singular Object type singular name.
      * @return void
      *
-     * @dataProvider getSetPluralizedProvider
-     * @covers ::_getPluralized()
-     * @covers ::_setPluralized()
+     * @dataProvider getSetSingularProvider
+     * @covers ::_getsingular()
```

Comment:
It should probably be `_getSingular()`. I'm not sure whether casing does matter to PHPUnit…